packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -338,6 +338,17 @@ export interface PluginConfigTypeIos {
    * and [Apple's documentation on Privacy manifest files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
    */
   privacyManifestAggregationEnabled?: boolean;
+
+  /**
+   * Enable support for prebuilt React Native iOS dependencies (ReactNativeDependencies.xcframework).
+   * This feature is only available from React Native 0.80.
```

Comment:
```suggestion
   * This feature is available from React Native 0.80.
```

The statement does not sound like future-proof. Soon, the React Native version will increment. Let's remove the word to avoid maintenance.